src/main/java/com/qaprosoft/carina/demo/gui/components/CreateAccountForm.java

```diff
@@ -0,0 +1,75 @@
+package com.qaprosoft.carina.demo.gui.components;
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractUIObject;
+import com.qaprosoft.carina.demo.gui.pages.SignUpPage;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.Random;
+
+public class CreateAccountForm extends AbstractUIObject {
```

Comment:
Maybe page?

Reply:
NO, we have SignUpPage and CreateAccountForm here

Reply:
> NO, we have SignUpPage and CreateAccountForm here

And why you decided to create separate CreateAccountForm component?Does it possible to interact with it on other pages?

Reply:
> > NO, we have SignUpPage and CreateAccountForm here
> 
> And why you decided to create separate CreateAccountForm component?Does it possible to interact with it on other pages?

I didn't know the components should be available on other pages. Ok, I going to change this.